vtex/faststore
# Feat: Enables override for `EmptyGallery` in `ProductGallery` section

## What's the purpose of this pull request?

- Enables override for `EmptyGallery` in `ProductGallery` section

## How it works?

- You can override `EmptyGallery` using `__experimentalEmptyGallery`.

## How to test it?

1. In `packages/core/src/customizations/src/components/overrides/ProductGallery.tsx`

change the following code:
```
const override: SectionOverride = {
  section: SECTION,
  components: { __experimentalEmptyGallery: { Component: () => <>Empty</> } },
}

```

2. run `yarn dev`
- search for 'a'
- You should be able to see "Empty" instead of the `EmtpyGallery` component.
<img width="1431" alt="image" src="https://github.com/vtex/faststore/assets/3356699/13084212-d172-4e42-bfcc-3378b74e02be">


### Starters Deploy Preview

https://github.com/vtex-sites/starter.store/pull/303
